### PR TITLE
Add ENV var to disable MWC init

### LINKED ISSUE #67: Allow MutatingWebhookConfiguration to be deployed separately
**What would you like to be added**:
Currently the webhook configuration is created/reconciled by `imageswap-init` from the template in the `imageswap-mwc` ConfigMap. It would be useful if we could deploy this resource separately to allow its lifecycle to be managed by an external tool (in our case, [Pulumi](https://www.pulumi.com/docs/get-started/kubernetes/)).

We have imported all resources from `deploy/install.yaml` into Pulumi, but when we use Pulumi to undeploy ImageSwap, the MutatingWebhookConfiguration remains, meaning that pods are denied by default since the controller is not there.

It would be great if ImageSwap could detect whether there is an existing webhook configuration present (which it does already), but have some mechanism to say "use it as-is; do not reconcile it".

**Why is this needed**:
We are onboarding ImageSwap and this is of a particular problem in our CI clusters where we apply Pulumi from a branch and then revert to the `main` branch afterwards, ready for the next branch build which may not install ImageSwap. It means pods from future runs are not admitted until we manually delete the webhook configuration.

### PR DESCRIPTION
**Pull request by phenixblue**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation

/kind feature
> /kind release

**What this PR does / why we need it**:

This adds a new `IMAGESWAP_DISABLE_AUTO_MWC` environment variable to allow for disabling the automatic generation of the Mutating Webhook Configuration (MWC) in the imageswap-init container.

This allows for the MWC to be generated outside of the imageswap init process and adds more flexibility to integrate cleanly with other provisioning tools/flows.

**Which issue(s) this PR fixes**:

Fixes #67

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
- Adds the `IMAGESWAP_DISABLE_AUTO_MWC` environment variable to allow for disabling the automatic generation of the Mutating Webhook Configuration (MWC) in the imageswap-init container.
```

**Additional documentation e.g., usage docs, etc.**:

```docs

```

### CODE REVIEW

File: deploy/install.yaml [hunk 1]
```diff
@@ -161,6 +161,7 @@ data:
   PYTHONUNBUFFERED: "TRUE"
   IMAGESWAP_MODE: "MAPS"
   IMAGESWAP_LOG_LEVEL: "INFO"
+  IMAGESWAP_DISABLE_AUTO_MWC: "TRUE"
```
Comment (ilrudie) [inline deploy/install.yaml:164]: **question:** do we want `IMAGESWAP_DISABLE_AUTO_MWC: "TRUE"` in this file if the default is meant to be false?

Comment (phenixblue) [reply deploy/install.yaml:164]: I think this was a manual change for testing while I was developing. I'm gonna go back and make sure, but likely I just missed reverting it.

Comment (phenixblue) [reply deploy/install.yaml]: Ok, this was leftover from testing, just cleaned it up

**Review by ilrudie — APPROVED** [review]
looks good to me

**Pull request merged**